src/cmd-buildupload

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3
+
+# The inverse of cmd-buildprep.
+
+import argparse
+import json
+import os
+import subprocess
+import sys
+import tempfile
+
+sys.path.insert(0, '/usr/lib/coreos-assembler')
```

Comment:
Can we use relative paths? e.g `sys.path.insert(0, os.path.dirname(os.path.abspath(__file__))`

Reply:
Good idea, fixed in the whole codebase!